### DISCUSSION TITLE
Failed to request create session

### DISCUSSION THREAD

**Discussion by D0nHuan** [ilyhalight/voice-over-translation#859] [Q&A]
не так давно началась проблема - такая ошибка на любом видео ютуба Failed to request create session
скрипт обновлён, использую Violetmonkey

**Comment by ilyhalight** [comment]
> скрипт обновлён

Какую версию используете?

Проверил с самой последней и все работает

**Reply by Kirillissimus** [reply]
Возможно ли это из-за блокировок IP Яндекса в регионе?

Перепроверил у себя - так и есть: без впн в моём регионе(Балтия) не работает. Причина - блокировка IPшников Яндекса.
https://yandex.ru/ips

**Reply by ilyhalight** [reply]
> Возможно ли это из-за блокировок IP Яндекса в регионе?
> 
> Перепроверил у себя - так и есть: без впн в моём регионе(Балтия) не работает. Причина - блокировка IPшников Яндекса. https://yandex.ru/ips

проверьте с [этой](https://github.com/ilyhalight/voice-over-translation/raw/refs/heads/dev/dist/vot.user.js) версией

**Reply by Kirillissimus** [reply]
> > Возможно ли это из-за блокировок IP Яндекса в регионе?
> > Перепроверил у себя - так и есть: без впн в моём регионе(Балтия) не работает. Причина - блокировка IPшников Яндекса. https://yandex.ru/ips
> 
> проверьте с [этой](https://github.com/ilyhalight/voice-over-translation/raw/refs/heads/dev/dist/vot.user.js) версией

Всё равно не работает. Только через vpn, но есть нюанс.

Проверил  через расширениие UrbanVPN для Хрома на сервере Германии и перевод видео удался. 
Отключился. Перезагрузил страницу Ютуба и попробовал снова включить перевод. Получил ошибку Failed to request create session.
Подключил снова ВПН, но уже на сервере из Эстонии(там же где и я).  Перезагрузил страницу Ютуба и перевод заработал. Очень странно.

Другими словами, на родном провайдете Elisa/Starman [EE] не работает, а на магистральном WaveCom[EE] отлично всё фурычит. Не знаю почему так.

**Reply by ilyhalight** [reply]
> > > Возможно ли это из-за блокировок IP Яндекса в регионе?
> > > Перепроверил у себя - так и есть: без впн в моём регионе(Балтия) не работает. Причина - блокировка IPшников Яндекса. https://yandex.ru/ips
> > 
> > 
> > проверьте с [этой](https://github.com/ilyhalight/voice-over-translation/raw/refs/heads/dev/dist/vot.user.js) версией
> 
> Всё равно не работает. Только через vpn, но есть нюанс.
> 
> Проверил через расширениие UrbanVPN для Хрома на сервере Германии и перевод видео удался. Отключился. Перезагрузил страницу Ютуба и попробовал снова включить перевод. Получил ошибку Failed to request create session. Подключил снова ВПН, но уже на сервере из Эстонии(там же где и я). Перезагрузил страницу Ютуба и перевод заработал. Очень странно.
> 
> Другими словами, на родном провайдете Elisa/Starman [EE] не работает, а на магистральном WaveCom[EE] отлично всё фурычит. Не знаю почему так.

Вероятно, все-таки, айпи яндекса в бане у вашего провайдера, но почему-то тестовый запрос (https://yandex.ru/favicon.ico) проходит нормально. Попытаюсь придумать какой-нибудь другой способ

**Reply by ilyhalight** [reply]
Вернул возможность ручного включения проксирования в меню расширения в [dev версии](https://github.com/ilyhalight/voice-over-translation/raw/refs/heads/dev/dist/vot.user.js#bypass=true). Проверьте с режимом "Proxy everything". Должно работать

**Reply by Kirillissimus** [reply]
> Вернул возможность ручного включения проксирования в меню расширения в [dev версии](https://github.com/ilyhalight/voice-over-translation/raw/refs/heads/dev/dist/vot.user.js#bypass=true). Проверьте с режимом "Proxy everything". Должно работать

Теперь всё работает! Огромная благодарность!! :)
